setup.py

```diff
@@ -85,8 +115,11 @@ class BuildExt(build_ext):
         'unix': [],
     }
 
+    UnixCCompiler.src_extensions.append(".mm")
+    UnixCCompiler.language_map[".mm"] = "objc"
```

Comment:
This ugly hack was necessary to support Objective-C

Reply:
I've seen worse :p